third_party/blink/renderer/platform/audio/audio_destination.cc

```diff
@@ -59,7 +65,11 @@ namespace {
 // that we would ever need. The current UMA stats indicates that this is, in
 // fact, probably too small. There are Android devices out there with a size of
 // 8000 or so.  We might need to make this larger. See: crbug.com/670747
+#if BUILDFLAG(IS_COBALT)
+constexpr uint32_t kFIFOSize = 128 * 128;
```

Comment:
Leave a TODO if we want to merge this PR. This issue will be gone once we rebase to m126+.

As the current Chromium uses `kFIFOSize = 128 * 128` and it uses the max of `callback_buffer_size_ + render_quantum_frames` and `kFIFOSize` as fifo_length.